packages/atlas/src/components/_overlays/BottomDrawer/BottomDrawer.styles.ts

```diff
@@ -0,0 +1,79 @@
+import styled from '@emotion/styled'
+
+import { ActionBar } from '@/components/ActionBar'
+import { cVar, zIndex } from '@/styles'
+
+export const DrawerOverlay = styled.div`
+  position: fixed;
+  z-index: ${zIndex.videoWorkspaceOverlay};
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: ${cVar('colorBackgroundOverlay')};
+  transition: opacity ${cVar('animationTransitionSlow')};
+
+  &.bottom-drawer-overlay {
+    &-enter-active,
+    &-exit {
+      opacity: 1;
+      pointer-events: initial;
+    }
+
+    &-enter,
+    &-exit-active {
+      opacity: 0;
+      pointer-events: none;
+    }
+  }
+`
+
+export const Container = styled.div`
+  transform: translateY(100%);
+  opacity: 0;
+  position: fixed;
+  z-index: ${zIndex.videoWorkspaceOverlay};
+  top: var(--size-topbar-height);
```

Comment:
this value cannot be fixed, e.g. in nft purchase drawer takes up the whole screen

Reply:
👍 I missed that. Added a `coverTopbar` prop that if true will make the drawer fill the full screen height